src/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Animated, View, ViewProps, ImageProps, Image } from 'react-native';
+import {
+  ScreenProps,
+  ScreenContainerProps,
+  ScreenStackProps,
+  ScreenStackHeaderConfigProps,
+  HeaderSubviewTypes,
+} from './types';
+
+export * from './types';
+
+let ENABLE_SCREENS = true;
+
+export function enableScreens(shouldEnableScreens = true): void {
+  ENABLE_SCREENS = shouldEnableScreens;
+}
+
+export function screensEnabled(): boolean {
+  return ENABLE_SCREENS;
+}
+
+export class NativeScreen extends React.Component<ScreenProps> {
+  render(): JSX.Element {
```

Comment:
Typing render method on class-based react component seems redundant.

Reply:
For some reason eslint warns if I remove it here, but it doesn't in `index.native.tsx`.